crates/sui-config/src/genesis.rs

```diff
@@ -298,18 +309,14 @@ impl Default for GenesisChainParameters {
 
 pub struct Builder {
     parameters: GenesisChainParameters,
+    // protocol version that the chain starts at.
+    protocol_version: ProtocolVersion,
```

Comment:
Maybe this should probably be put into the parameters type that way it is clear what version is being targeted (when looking at the genesis repo) since the parameters are all serialized on disk during the ceremony

Reply:
fixed